src/testing.ts

```diff
@@ -276,12 +277,12 @@ export function runTests(force: boolean, onlyResults: boolean, unsafe = false, e
 }
 
 function preTestChecks(): boolean {
-  if (!checkNoCairo('example__contracts')) {
-    console.log('Please remove example__contracts, or run with -f to delete it');
+  if (!checkNoCairo(path.join('warp_test', 'example_contracts'))) {
+    console.log('Please remove example_contracts, or run with -f to delete it');
```

Comment:
Should probably read `warp_test/example_contracts` instead

Reply:
why?

Reply:
Because that's the folder that should be removed

Reply:
right I see, I was confused between the code and string literal